platform/ios/src/MGLAnnotationView.mm

```diff
@@ -239,7 +239,10 @@ - (void)setDragState:(MGLAnnotationViewDragState)dragState animated:(BOOL)animat
     }
     else if (dragState == MGLAnnotationViewDragStateCanceling)
     {
-        NSAssert(self.annotation, @"Annotation property should not be nil.");
+        if (!self.annotation) {
+            [NSException raise:@"NSInvalidArgumentException"
```

Comment:
[`NSInvalidArgumentException`](https://developer.apple.com/reference/foundation/nsinvalidargumentexception) is a constant.